atlasdb-config/src/test/java/com/palantir/atlasdb/factory/startup/TimeLockMigratorTest.java

```diff
@@ -122,73 +71,47 @@ public void invalidationDoesNotProceedIfTimelockPingUnsuccessful() {
     public void migrationDoesNotProceedIfInvalidationFails() {
         when(invalidator.backupAndInvalidate()).thenThrow(new IllegalStateException());
 
-        TimeLockMigrator migrator =
-                TimeLockMigrator.create(
-                        MetricsManagers.createForTests(),
-                        timelockConfig.toNamespacedServerList(), invalidator, USER_AGENT);
+        TimeLockMigrator migrator = TimeLockMigrator.create(timestampManagementService, invalidator);
         assertThatThrownBy(migrator::migrate).isInstanceOf(IllegalStateException.class);
-        wireMockRule.verify(0, postRequestedFor(urlEqualTo(TEST_ENDPOINT)));
+        verify(timestampManagementService, times(0)).fastForwardTimestamp(anyLong());
```

Comment:
nit: usually we use `never()` rather than `times(0)` for readability - they're equivalent